### PR TITLE
technical document arrange

### PR DESCRIPTION
**Pull request by cmcm2222**
### Background
Sebak wiki's contents have a lot of changes. 
so I check all of contents with @soonkuk.

### CODE REVIEW

**Comment by spikeekips** [comment]
And if possible, I wish the `docs` moves to new repository.

File: docs/[tech_doc]how_the_consensus_protocol_works.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+   * If the node is not proposer, it waits for and receive the B(`INIT`, `YES`).
+      * When it receives the ballot, the node goes to the next state.
+   * When the timer expires, the node goes to the next state.
+
+### `SIGN`
+1. The timer is reset to TIMEOUT_SIGN
+1. The node checks [the proposed ballot is valid](./[tech_doc]how_to_check_a_ballot_is_valid.md).
+   * If the proposed ballot is valid, the node broadcasts B(`SIGN`, `YES`).
+   * If the proposed ballot is invalid, the node broadcasts B(`SIGN`, `NO`).
+   * If the proposed ballot is empty, the node broadcasts B(`SIGN`, `EXP`).
```
Comment (soonkuk) [inline docs/[tech_doc]how_the_consensus_protocol_works.md]: @cmcm2222 I checked this with @Charleslee522 
`If the proposed ballot is empty, the node broadcasts B(`SIGN`, `EXP`)` should be changed to
`If the proposed ballot is empty, the node broadcasts B(`SIGN`, `YES`)`

File: docs/sebak_Installation.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+## Prerequisite
```
Comment (Charleslee522) [inline docs/sebak_Installation.md]: File name should be sebak_installation.md

File: docs/sebak_deployment.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+docker rmi -f $(docker images -f "dangling=true" -q)
+```
+
+Then you can spawn 3 nodes using the following commands:
+
+```
+docker run --net host --rm -it --env-file=docker/node1.env sebak
+docker run --net host --rm -it --env-file=docker/node2.env sebak
+docker run --net host --rm -it --env-file=docker/node3.env sebak
+```
```
Comment (Charleslee522) [inline docs/sebak_deployment.md]: Need new line??

File: docs/sebak_deployment_standalone.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
+By the nature of design, SEBAK should be deployed as composition of nodes and they should tightly connected for consensus. This standalone mode recommend only for testing and debugging. With standalone mode, you do not need to compose quorum and another server. It does not make consensus, just execute incoming messages including transactions, but it do the right work, such as inflation, collecting transaction fee, etc.
+
+## Installation
+
+Please follow the [Installation](./sebak_Installation.md). If you are so busy and you already have golang (1.11 or higher) environment, do like this,
```
Comment (Charleslee522) [inline docs/sebak_deployment_standalone.md]: File name link(sebak_Installation.md) should be also modified.

File: docs/sebak_command.md [hunk 5]
```diff
@@ -0,0 +1,168 @@
+To install `sebak` command, please check the [installation guide](./sebak_Installation.md).
```
Comment (Charleslee522) [inline docs/sebak_command.md]: File name link(sebak_Installation.md) should be also modified.

File: docs/sebak_deployment.md [hunk 6]
```diff
@@ -0,0 +1,192 @@
...
+	tls-key=sebak.key
+	log-level=debug
+	log-format=terminal
+	log=
+....
+```
+
+If nodes launched sucessful, each node will connect other nodes and will be ready to make consensus.
+
+# Spinning Network using Docker
```
Comment (Charleslee522) [inline docs/sebak_deployment.md]: I figure, using docker scenario is simpler than command line.
How about putting it up? It’s just my personal opinion.

**Comment by anarcher** [comment]
I have a question the filename of `docs/[tech_doc]how_the_consensus_protocol_works.md`. 
I think that it need some categories. and `[tech_doc]` prefix of the file name is odd IHMO.

File: docs/[tech_doc]how_the_consensus_protocol_works.md (hunk 1 above)
Comment (cmcm2222) [reply docs/[tech_doc]how_the_consensus_protocol_works.md]: OK. I will change.

File: docs/sebak_Installation.md (hunk 2 above)
Comment (cmcm2222) [reply docs/sebak_Installation.md]: thanks :)

File: docs/sebak_deployment.md (hunk 3 above)
Comment (cmcm2222) [reply docs/sebak_deployment.md]: nope.

**Comment by cmcm2222** [comment]
@anarcher  I understand. 
I will change file prefix.

File: docs/sebak_deployment.md (hunk 6 above)
Comment (cmcm2222) [reply docs/sebak_deployment.md]: I understand. 
But I leave this phrase as it is. 
thanks for advice. :)

File: docs/sebak_deployment_standalone.md (hunk 4 above)
Comment (cmcm2222) [reply docs/sebak_deployment_standalone.md]: I changed it.

**Comment by cmcm2222** [comment]
Ping @anarcher @Charleslee522 @soonkuk

File: docs/sebak_command.md (hunk 5 above)
Comment (cmcm2222) [reply docs/sebak_command.md]: I changed it.

**Comment by cmcm2222** [comment]
@Charleslee522, @soonkuk  Please confirm review to merge.

**Pull request merged**